docs/source/en/model_doc/clip.mdx

```diff
@@ -75,6 +75,25 @@ encode the text and prepare the images. The following example shows how to get t
 
 This model was contributed by [valhalla](https://huggingface.co/valhalla). The original code can be found [here](https://github.com/openai/CLIP).
 
+## Resources
+
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with CLIP. If you're
+interested in submitting a resource to be included here, please feel free to open a Pull Request and we will review it.
+The resource should ideally demonstrate something new instead of duplicating an existing resource.
+
+<PipelineTag pipeline="text-to-image"/>
+- A blog post on [How to use CLIP to retrieve images from text](https://huggingface.co/blog/fine-tune-clip-rsicd).
+- A blog bost on [How to use CLIP for Japanese text to image generation](https://huggingface.co/blog/japanese-stable-diffusion).
```

Comment:
The first sentence should be something along the lines of "How to fine-tune CLIP on image-text pairs", the second one is not relevant for CLIP, as it's a blog about Stable Diffusion.